.github/workflows/wine-arch.yml

```diff
@@ -0,0 +1,29 @@
+name: Wine Archlinux CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: archlinux:latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Compilation
+        run: |
+          echo -e "[multilib]\nInclude = /etc/pacman.d/mirrorlist" >> /etc/pacman.conf
+          yes|pacman -Syu --noconfirm base-devel sudo
+          useradd user -G wheel && echo "user ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+          chown user -R . && cd wine-tkg-git
+          su user -c "yes|PKGDEST=/tmp/wine-tkg makepkg --noconfirm -si"
```

Comment:
Unnecessary `yes`?

Reply:
Doesn't works without `yes`.

Reply:
Deleted all `yes` in Arch CI.

Reply:
I thought about it some more; this one is actually probably needed. I forgot that the `PKGBUILD` is interactive; you probably will need this particular one in order to select the default values for the TkG configuration.

Reply:
Done.